edit_course/operations/configure.py

```diff
@@ -213,7 +222,16 @@ def update_learning_objects( # noqa: MC0001
                 if obj_key in o:
                     setattr(lobject, key, parse_bool(o[obj_key]))
 
-        if lobject_cls in (LTIExercise, BaseExercise):
+        if lobject_cls == LTI1p3Exercise:
+            lobject.lti_service = LTI1p3Service.objects.filter(menu_label=str(o["lti1p3"])).first()
+            obj_key = "lti_custom"
+            if obj_key in o:
+                setattr(lobject, key, o[obj_key])
+            obj_key = "lti_open_in_iframe"
+            if obj_key in o:
+                setattr(lobject, key, parse_bool(o[obj_key]))
```

Comment:
This isn't right. The `key` is never assigned here. Something like
```
key = "open_in_iframe"
if f"lti_{key}" in o:
    setattr(lobject, key, parse_bool(o[f"lti_{key}"]))
```
or just
```
if "lti_open_in_iframe" in o:
    setattr(lobject, "open_in_iframe", parse_bool(o["lti_open_in_iframe"]))
```
would work (same with lti_custom)

Reply:
Oops... Now it should be right